doc/classes/AudioEffectGate.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AudioEffectGate" inherits="AudioEffect" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
```

Comment:
I'm curious whether this should be called AudioEffectNoiseGate, so that searching for "noise" in the class reference lets you find this effect.